test/api/unit/models/organization.test.js

```diff
@@ -205,6 +205,20 @@ describe('Organization model', () => {
     });
   });
 
+  describe('.byName()', () => {
+    it('returns organizations ordered by name', async () => {
+      const orgB = Organization.create({ name: 'Org B' });
+      const orgA = Organization.create({ name: 'Org A' });
+      const orgC = Organization.create({ name: 'Org C' });
```

Comment:
Would you consider testing the sorting with strings that also start with numbers? Thinking about where "10x" and 18F" go

Reply:
The sorting happens in the underlying database via ORM in the Organization model — I expect it to be ASCIIbetical: 
```js
  Organization.addScope('byName', {
    order: [
      ['name', 'ASC'],
    ],
  });
```

It's the job of this unit test to test the organization model and in this specific case what it's doing is verifying that the sorting by org name is being _applied_ to these results, not verifying the details of the sorting algorithm. I would absolutely consider testing other aspects of sorting if we have a good reason to do that — and you may be seeing one that I haven't! — but my gut tells me that wouldn't come up unless we found ourselves writing code to customize the sort to address a particular need.